run-upgrade-tests.sh

```diff
@@ -61,6 +71,8 @@ pifpaf_stop(){
 cleanup(){
     pifpaf_stop
```

Comment:
You have `pifpaf_stop` here not sure it works if `indexer_stop` works?
I mean there's too many stop begin called now I think :)

Reply:
I don't think so, the pifpaf_stop should still be the non-prefixed service running under pifpaf which is Gnocchi, while indexer_stop is, well, the indice database :)